### DISCUSSION TITLE
Arabba-SR13 -> Borca-SR1 should not be considered a patch update

### DISCUSSION THREAD

**Discussion by breun** [renovatebot/renovate#15831] [Suggest an Idea]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.71.0

### Please select which platform you are using if self-hosting.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

14.8.5-ee

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

A Maven project using `io.r2dbc:r2dbc-bom:Arabba-SR13` with major and minor update disabled in the Renovate config, will receive a merge request to update from `Arabba-SR13` to `Borca-SR1`. I think changing to a different release train should be considered a major update instead of a patch update.

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
Not a bug, converted to discussion.

Renovate cannot possibly be expected to automatically understand every fun naming scheme projects could use, when they choose not to use numbers.

In rare exceptions for widely used schemes like Ubuntu and Debian, we add custom versioning, but (a) it's a feature request, (b) it adds complexity to our code base, so it needs to be justified by its volume.

**Reply by breun** [reply]
I understand there are a lot of weird versioning schemes, but why does Renovate consider this particular update a patch update? Maven’s own artifact version parsing would require two dots in the version for that, and I believe that’s a pretty standard way of defining patch updates.
